tezos_interop/tezos_interop.rei

```diff
@@ -153,7 +157,8 @@ module Consensus: {
   let listen_operations:
     (~context: Context.t, ~on_operation: operation => unit) => unit;
   let fetch_validators:
-    (~context: Context.t) => Lwt.t(result(list(Key.t), string));
+    (~context: Context.t) =>
+    Lwt.t(result(list(Crypto.Ed25519.Key.t), string));
```

Comment:
I think this should continued being `list(Key.t)`. Validators can have the new secp256k1 keys cant they, @EduardoRFS ?